lasagne/layers/conv.py

```diff
@@ -117,23 +109,22 @@ class Conv1DLayer(Layer):
         An integer or a 1-element tuple specifying the stride of the
         convolution operation.
 
-    border_mode : str, one of 'valid', 'full', 'same'
-        A string indicating the convolution border mode.
+    pad : {0, (0,), 'full', 'same'} (default: 0)
```

Comment:
> why not? it looks like it's still in the code below...

Yes, we decided we'd keep it for compatibility reasons (people having the border mode as a positional argument), but have the official way be `pad=0`. Hmm, thinking about it, I guess we actually decided to give a warning in that case, but Martin didn't and I didn't remember to add it now.